index.html

```diff
@@ -9,56 +9,52 @@
 
   <body>
     <header>
-
       <nav>
-       <ul class="left">
-         <li><img class="shoelogo" src="logo.jpg" alt="shoelogo"></li>
-         <li><a href="#">MEN</a></li>
-         <li><a href="#">WOMEN</a></li>
-         <li><a href="#">BOYS</a></li>
-         <li><a href="#">GIRLS</a></li>
-         <li><a href="#">CUSTOMIZE</a></li>
-       </ul>
-       <ul class="right">
-         <li><a href="#">HELP</a></li>
-         <li><a href="#">MY ACCOUNT</a></li>
-         <li><a href="#"><img src="cart.jpg" alt="shopping cart logo">shopping cart (2)</a></li>
-      </ul>
+        <ul class="left">
+           <li><img class="shoelogo" src="logo.jpg" alt="shoelogo"></li>
+           <li><a href="#">MEN</a></li>
+           <li><a href="#">WOMEN</a></li>
+           <li><a href="#">BOYS</a></li>
+           <li><a href="#">GIRLS</a></li>
+           <li><a href="#">CUSTOMIZE</a></li>
+         </ul>
+
+         <ul class="right">
+           <li><a href="#">HELP</a></li>
+           <li><a href="#">MY ACCOUNT</a></li>
+           <li><a href="#"><img src="cart.jpg" alt="shopping cart logo">shopping cart (2)</a></li>
+        </ul>
+      </nav>
```

Comment:
There were a few closing tags missing in here, like this `</nav>`, that I added. This makes sure things are where you expect them to be.